sdk/tables/data-tables/src/utils/continuationToken.ts

```diff
@@ -5,26 +5,26 @@ import { base64Decode, base64Encode } from "./bufferSerializer";
 
 export interface ContinuationToken {
   nextPartitionKey: string;
-  nextRowKey: string;
+  nextRowKey?: string;
 }
 
 /**
  * Encodes the nextPartitionKey and nextRowKey into a single continuation token
  */
 export function encodeContinuationToken(
-  nextPartitionKey: string = "",
-  nextRowKey: string = ""
+  nextPartitionKey?: string,
+  nextRowKey?: string
 ): string | undefined {
-  if (!nextPartitionKey && !nextRowKey) {
+  if (!nextPartitionKey) {
     return undefined;
   }
 
-  const continuationToken = JSON.stringify({
+  const continuationToken: { nextPartitionKey: string; nextRowKey?: string } = {
```

Comment:
I'm not sure why it would make a difference, the inferred type is basically the same (though it's nextRowKey being optional rather than maybe undefined)

<img width="222" alt="image" src="https://user-images.githubusercontent.com/639216/158909735-8ca45a81-ba99-4b11-9e45-e6db5a50e338.png">

Certainly JSON.stringify does not care about its input. :)

Reply:
Yeah not really needed, I removed it